Makefile

```diff
@@ -0,0 +1,2 @@
+flake: 
+	flake8 --exclude migrations powerdns/
```

Comment:
Mostly it's empty files, like:
https://github.com/allegro/django-powerdns-dnssec/blob/develop/dnsaas/admin.py
https://github.com/allegro/django-powerdns-dnssec/blob/develop/dnsaas/tests.py
which dont pass flake8
I can fix flake8 (which results having above files empty) or rm those files from repo.